README.md

```diff
@@ -34,13 +89,20 @@ Feel free to use it and extend it. We are willing to hear about your suggestions
 
 
 
-## Installation
+## External Resources
```

Comment:
It may be better to place the "External Resources" section at the end of the Readme